packages/frontend/src/components/wallet-migration/WalletMigration.jsx

```diff
@@ -94,11 +95,9 @@ const WalletMigration = ({ open, onClose }) => {
             listOfAccounts: accounts.join(', '),
             selectedWallet: walletName,
         });
-        flushEvents()
-            .finally(() => {
-                setMigrationStep(WALLET_MIGRATION_VIEWS.VERIFYING);
-                handleSetActiveView(WALLET_MIGRATION_VIEWS.REDIRECTING);
-            });
+        flushEvents();
```

Comment:
At the moment, when `flushEvents()` gets called we will have descrepencies in our data as `flushEvents()` calls `rudderStack.reset()` which will reset the current logged in user data. We should remove flushEvents() here.